### ISSUE TITLE
`tooltip` property on disabled DropdownItem does not work

### ISSUE THREAD

**Issue by KKoukiou** [patternfly/patternfly-react#4581]
`tooltip` property on disabled DropdownItem does not work

``` jsx
      <DropdownItem key="disabled link" tooltip={<Tooltip>test</Tooltip>} isDisabled description="Disabled link description">
        Disabled Link
      </DropdownItem>
```

So the above code snippet will not show any tooltip.

Probably related to #1894

**Comment by tlabaj** [comment]
@KKoukiou can you please tell me what version of patternFly React you are using.

**Comment by KKoukiou** [comment]
> @KKoukiou can you please tell me what version of patternFly React you are using.

@patternfly/react-core": "3.158.1",

**Comment by KKoukiou** [comment]
> > @KKoukiou can you please tell me what version of patternFly React you are using.
> 
> @patternfly/react-core": "3.158.1",

Same happends with "@patternfly/react-core": "4.32.1",

**Comment by jschuler** [comment]
That is probably to be expected, since disabled elements do not emit hover events. I believe @seanforyou23 did some research in this area and found a solution for another component

**Comment by jschuler** [comment]
Here's one possible workaround in the meantime:
Give your dropdown item an ID:
```
<DropdownItem id="disabledItem" key="disabled link" isDisabled description="Disabled link description">
        Disabled Link
      </DropdownItem>,
```
Since "@patternfly/react-core" 4.32.8 Tooltip has a `reference` prop you can use:
```
<Tooltip content="test" reference={() => document.getElementById('disabledItem')} />
```
Then you should always get the tooltip

https://codesandbox.io/s/restless-night-s2mze?file=/index.js

**Comment by KKoukiou** [comment]
> Here's one possible workaround in the meantime:
> Give your dropdown item an ID:
> 
> ```
> <DropdownItem id="disabledItem" key="disabled link" isDisabled description="Disabled link description">
>         Disabled Link
>       </DropdownItem>,
> ```
> 
> Since "@patternfly/react-core" 4.32.8 Tooltip has a `reference` prop you can use:
> 
> ```
> <Tooltip content="test" reference={() => document.getElementById('disabledItem')} />
> ```
> 
> Then you should always get the tooltip
> 
> https://codesandbox.io/s/restless-night-s2mze?file=/index.js

Thanks for the workaround, it also works if you wrap the component with the tooltip instead of passing the tooltip prop:

```
                <Tooltip content='test'>
                    <DropdownItem isDisabled>
                       Delete
                    </DropdownItem>
                </Tooltip>

```

**Comment by tlabaj** [comment]
this issue will be cover by issue #5286

**Comment by cben** [comment]
Dup of #5826 (not 5286).  Solved now — use latest packages to include #6038 and use `isAriaDisabled` instead of `isDisabled`.  #6109 remains about screen readers but not related to disabling.

(Similar support for `isAriaDisabled` is available on some other components e.g. Button, ref #1894 for the big picture)
